de.metas.procurement.base/src/main/java/de/metas/procurement/base/order/impl/PMMPurchaseCandidateBL.java

```diff
@@ -57,7 +57,7 @@ public void subtractQtyOrdered(final I_PMM_PurchaseCandidate candidate, final Bi
 	@Override
 	public void updateQtyToOrderFromQtyToOrderTU(final I_PMM_PurchaseCandidate candidate)
 	{
-		final I_M_HU_PI_Item_Product huPIItemProduct = candidate.getM_HU_PI_Item_Product();
+		final I_M_HU_PI_Item_Product huPIItemProduct = getM_HU_PI_Item_Product_Effective(candidate);
```

Comment:
now that i am looking at this change i am wondering why not calling this method from model interceptor and/or callout when user is changing the M_HU_PI_Item_Product_Override.

Use case is: (keep in mind that purchasers are always using the TU quantities)
Purchaser is changing the M_HU_PI_Item_Product_Override_ID so the Qty of CUs/TU is also changed.
In this case i guess we need to update ALL the CU quantities.
wdyt?